spec/support/contracts/json_schema_test/app/publish.mson

```diff
@@ -25,7 +25,7 @@ A post that only has a title, but no text body. Weird, right?
 - body: (string) - Body of the post
 - author_id: 2 (number, required) - User id of author
 - primary_tag: (App::Tag, optional) - Property of custom type
-- tags: (array[Tag]) - An array with a custom type
+- tags: (array[App::Tag]) - An array with a custom type
```

Comment:
@dgmora I think there was a misunderstanding. What you did:

```markdown
# App:Object1
- id: (number)

# App::Object2
- foo: (array[App::Object1])
```

and not

```markdown
# App::Object2
- foo: (array[Object1])
```

That is totally correct, and if this worked before than that was a bug. 

What I thought you meant, because I didn't take a proper look, is: the following is not valid, only `# App::LocalObject` is supported, but not:

```markdown
# LocalObject
- id: (number)
```

So all is good :)